openshift/rhmap-nginx-proxy.json

```diff
@@ -0,0 +1,186 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+
+        "name": "nginx-proxy",
+        "description": "RHMAP proxy",
+        "tags": "rhmap,mobile,nginx,proxy"
+    },        
+    "parameters": [
+        {
+            "name": "BASE_HOST",
+            "value": "http://localhost",
+            "description": "OpenShift router domain that would be used to proxy to subdomains",
+            "required": true
+        },{
+            "name": "RHMAP_PLATFORM_PATH",
```

Comment:
After investigation we found numerous endpoints that needede to be exposed for apps. All of them are in `/box/srv/1.1` root so I'm using this as simplification.